packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -74,13 +74,17 @@ export function CreatePatternModalContents( {
 
 	const { categoryMap, findOrCreateTerm } = useAddPatternCategory();
 
+	const validation = useValidation();
```

Comment:
There's another call to `isItemValid`. Do you think we can also update it in this PR and so remove the `isItemValid` utility in favour of this approach? It seems mostly a one-liner, but happy to defer to a follow-up if it isn't.